app/src/main/res/values/strings.xml

```diff
@@ -49,4 +49,8 @@
     <string name="notification_title">Notification Title</string>
     <string name="action_dumb_login">Sign in with Device ID</string>
     <string name="dumb_register_txt">Sign up with Device ID</string>
+    <string name="lottery_guidelines_button">Selection Details</string>
+    <string name="lottery_guidelines_dialog_title">How this lottery works</string>
+    <string name="lottery_guidelines_dialog_message">This is placeholder text describing how entrants are chosen during the lottery selection. Update this message with the final guidelines for your event.</string>
```

Comment:
> The lottery selection process uses a fair distribution random selection algorithm, namely "Fisher–Yates shuffle". It is run when the selection deadline is reached. Firstly, it uses this algorithm to shuffle the input set, which is the set of all entrants. Next, it picks the first N elements from the resulting list, where N is the number of seats in the event, as set by the organizer.
>
> By default, all events have an unlimited number of entrants - that is, people can keep joining the entrant list. The organizer may decide to limit the maximum number of entrants allowed in the waiting list. If this is the case, you'll see the limit in the event details page.
>
> Regardless of this option, the input set for shuffle is always the whole waiting list at the time of selection.

Reply:
Also worth checking if there's a way to hyper link to the wiki page for Fisher-Yates in that text.